src/components/Glossary/GlossaryTooltip/index.tsx

```diff
@@ -13,7 +13,15 @@ const GlossaryTooltip = ({ children, termKey }: GlossaryTooltipProps) => {
   const isLargeScreen = useBreakpointValue({ base: false, lg: true })
 
   return isLargeScreen ? (
```

Comment:
We use desktop menu at 768px wide, but tooltip is only available at 992px wide. Is this intentional @pettinarip @konopkja?

Reply:
no, i think the original component was just built for desktop, but we want it make it available on on all screen resolutions.

I believe there is a PR to migrate the tooltip to chakra component first tho. Maybe pablo can shed more light on it.

Reply:
Correct. We will enable the tooltip for all the resolutions once we migrate our current component to Chakra (#10974).